armi/reactor/converters/geometryConverters.py

```diff
@@ -760,7 +773,7 @@ def _createRadialThetaZone(
             for nuc in self._sourceReactor.blueprints.allNuclidesInProblem:
                 material.setMassFrac(nuc, 0.0)
             newComponent = components.DifferentialRadialSegment(
-                "mixture", material, **dims
+                f"component {newBlockType}", material, **dims
```

Comment:
did you mean `material.name` here?

Reply:
No, I didn't mean to update the material name. I'll revert. I want the component to inherit the same type as the block so that the flags and type are set based on the block.